tests/datatypes/test_list.py

```diff
@@ -686,7 +686,6 @@ class InplaceListOperationTests(InplaceOperationTestCase, TranspileTestCase):
         'test_add_frozenset',
         'test_add_range',
         'test_add_set',
```

Comment:
We have a way to compensate for this - if you look at [test_complex.py](https://github.com/pybee/voc/blob/master/tests/datatypes/test_complex.py#L50), you'll see a "substitutions" dictionary declared in the test. This dictionary defines all the possible ways that a result might be rendered, and provides alternatives. Any test output will be normalised using these substitutions - any string found in the list of values will be replaced with the key value from the dictionary. So, pick one possible output ordering from the set.  enumerate all other possible outputs as alternatives, and the test should pass.